### DISCUSSION TITLE
What is the best framework have you used before ?

### DISCUSSION THREAD

**Discussion by Gayan-Sachintha** [community/community#135878] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What is the best framework have you used before ? 
Give the reasons for it also

**Comment by Gayan-Sachintha** [comment]
React

**Reply by MalsriManaram** [reply]
flutter

**Reply by Gayan-Sachintha** [reply]
Thank you

**Comment by hirun-kolongahapitiya** [comment]
Flutter 
Laravel

**Reply by Gayan-Sachintha** [reply]
Thank you

**Comment by Gayan-Sachintha** [comment]
Laravel 8.0

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/135879#discussioncomment-10349336) for reference to why this discussion was closed and locked. Thanks!
